memgpt/cli/cli.py

```diff
@@ -615,6 +615,15 @@ def run(
         # create agent
         try:
             preset = ms.get_preset(preset_name=agent_state.preset, user_id=user.id)
+            if preset is None:
```

Comment:
Without this I think the user will get an error until they run `memgpt configure` when going from 0.3.x -> 0.3.3